### ISSUE TITLE
VLC duplicate installation creates error

### ISSUE THREAD

**Issue by DanFGC** [freegeekchicago/fgc-installscript#67]
VLC is already installed by default in Mint. When we install it again by running the script this creates an error which then needs to be fixed. Removing VLC from install will fix this.

**Comment by steenburgh** [comment]
Fair enough. Feel free to submit a pull request to fix this. Otherwise I'll take care of it.

**Comment by steenburgh** [comment]
Nevermind I just went ahead and did it. Was easy enough that I could just do it from the web ui.

**Comment by steenburgh** [comment]
@ahanson Please close this issue as it's not resolved.
